docs/notebooks/01_merge.ipynb

```diff
@@ -1,27 +1,564 @@
 {
```

Comment:
<p>I'd briefly mention what <code>validate_obs</code> does. You can reference function documentation in sphinx using</p><pre class="ql-syntax" spellcheck="false">{func}<code>~atlas_protocol_scripts.pp.validate_obs</code>
</pre><p> </p><p><code>~</code> leads to only the function name being displayed rather than the full path.</p><p> </p><p>EDIT: I hate reviewnb. You need to put the function name in backticks, but they don't render in this comment. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
with the proper formatting (outside reviewnb): 

```myst
{func}`~atlas_protocol_scripts.pp.validate_obs`
```